packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -76,7 +70,7 @@ export interface ClipboardCopyProps extends Omit<React.HTMLProps<HTMLDivElement>
   exitDelay?: number;
   /** Delay in ms before the tooltip appears. */
   entryDelay?: number;
-  /** Delay in ms before the tooltip message switch to hover tip. */
+  /** @deprecated Delay in ms before the tooltip message switch to hover tip. */
```

Comment:
Why is this being deprecated?  Was its only purpose for the Code editor.  Could consumers be relying on it?

Reply:
It's being deprecated because of a subtle change in how the three components with a copy feature are handling toggling their tooltip text between 'copy' and 'copied' (or some similar variant).

Before this change, each of the three copy components were handling it differently. CodeBlock was never changing its 'copied' text back to 'copy'. CodeEditor had it hardcoded so that the text changed after 2500 milliseconds. And ClipBoardCopy was using this switchDelay value. 

In the later two cases, these timers caused some weird cases where the 'copied' text would switch back before the tooltip closed and cause the text to quickly update before it closed (sometimes with barely enough time for a user to read it, and that is a bit jarring when it happens). I spent some time trying to tweak timers to figure out how to avoid it, but ultimately, since the user can keep the tooltip open for an indeterminate amount of time by interacting with the tooltip's toggle, it was never going to work ideally with timers.

My solution was to wire all three components to trigger the switch between the 'copied' and 'copy' text only when the close transition has completed. That's now handled internally by the function passed to the `onTooltipHidden` on line 210 of clipboardCopy.tsx. So the switchDelay prop no longer does anything in ClipboardCopy.